doc/examples/with_widgets_overlay/lib/example_game.dart

```diff
@@ -12,26 +12,20 @@ class ExampleGame extends Game with HasWidgetsOverlay, TapDetector {
 
   @override
   void render(Canvas canvas) {
-    canvas.drawRect(const Rect.fromLTWH(100, 100, 100, 100),
-        Paint()..color = BasicPalette.white.color);
+    canvas.drawRect(
+      const Rect.fromLTWH(100, 100, 100, 100),
+      Paint()..color = BasicPalette.white.color,
+    );
   }
 
   @override
   void onTap() {
     if (isPaused) {
-      removeWidgetOverlay('PauseMenu');
+      overlays.remove('PauseMenu');
```

Comment:
how about show and hide? it feels more close to the behavior than add/remove. but I am ok with either

Reply:
Show and hide would give the wrong idea that only the visibility of the overlays would change when they are actually completely removed from the widget tree.